component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/internal/EmailOTPAuthenticatorServiceComponent.java

```diff
@@ -22,22 +22,18 @@
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.*;
```

Comment:
Please specify the required imports instead of importing all packages?

Reply:
Fixed in https://github.com/wso2-extensions/identity-outbound-auth-email-otp/pull/93